src/harness/harness.ts

```diff
@@ -1485,67 +1517,95 @@ namespace Harness {
             return;
 
             function checkBaseLines(isSymbolBaseLine: boolean) {
-                const fullBaseLine = generateBaseLine(fullResults, isSymbolBaseLine);
-
                 const fullExtension = isSymbolBaseLine ? ".symbols" : ".types";
-
                 // When calling this function from rwc-runner, the baselinePath will have no extension.
                 // As rwc test- file is stored in json which ".json" will get stripped off.
                 // When calling this function from compiler-runner, the baselinePath will then has either ".ts" or ".tsx" extension
                 const outputFileName = ts.endsWith(baselinePath, ts.Extension.Ts) || ts.endsWith(baselinePath, ts.Extension.Tsx) ?
-                    baselinePath.replace(/\.tsx?/, fullExtension) : baselinePath.concat(fullExtension);
-                Harness.Baseline.runBaseline(outputFileName, () => fullBaseLine, opts);
+                    baselinePath.replace(/\.tsx?/, "") : baselinePath;
+
+                if (!multifile) {

```

Comment:
do we still need the non-multifile mode? why not get rid of it?

Reply:
We use the non-multifile mode for the compiler runner, still. Which is probably a good thing, since those tests are small.